DownFall/Extensions/UIColor+Extensions.swift

```diff
@@ -31,4 +31,5 @@ extension UIColor {
     static let storeBlack = UIColor(rgb: 0x171717)
     static let highlightGold = UIColor(rgb: 0xfcd833)
     static let highlightGreen = UIColor(rgb: 0x2a4f36)
+    static let menuPurple = UIColor(rgb: 0x4c2f84)
```

Comment:
Can we collect any other random colors and add them to this before checking this in?